testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -189,6 +192,74 @@ impl K8sSwarm {
         k8snode.start().await?; // actually start the node. if port-forward is enabled, this is when it gets its ephemeral port
         Ok((peer_id, k8snode))
     }
+
```

Comment:
it would be great if we could unittest new functionality, although the existing unittests are very few

Reply:
Added a unit test for this